docs/standalone/no_attestation.sh

```diff
@@ -16,9 +16,16 @@
 set -e
 
 # Create a public and private key pair.
-gpg --quick-generate-key --yes attestor@example.com
-gpg --armor --export attestor@example.com > gpg.pub
-gpg --armor --export-secret-keys attestor@example.com > gpg.priv
+GPG_OUTPUT="$(gpg --quick-generate-key --yes attestor@example.com)"
+
+# Save its fingerprint.
+KEY_FINGER_PRINT="$(echo $GPG_OUTPUT | sed -n 's/.*\([A-Z0-9]\{40\}\).*/\1/p')"
```

Comment:
Could we use `gpg --fingerprint > fingerprint` instead here?

Reply:
If we do something like `gpg --fingerprint attestor@example.com`. Then it will have the same problem as before.